sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormLine.java

```diff
@@ -47,7 +47,7 @@ public FormLine(String text, BoundingBox boundingBox, Integer pageNumber,
      * {@inheritDoc}
      */
     @Override
-    public BoundingBox getBoundingBox() {
+    public FieldBoundingBox getBoundingBox() {
```

Comment:
`getFieldBoundingBox()`?

Reply:
We have actually decided to keep this as `getBoundingBox()`